float32 stamp
uint32 encoderR
uint32 encoderL
bool chksum